src/os/ObjectStore.h

```diff
@@ -393,7 +393,10 @@ class ObjectStore {
       __le32 oid;
       __le64 off;
       __le64 len;
-      __le32 dest_cid;
+      union {
+	__le32 fadvise_flags; 		//OP_WRITE
+	__le32 dest_cid;
+      };
       __le32 dest_oid;                  //OP_CLONE, OP_CLONERANGE
```

Comment:
Why fadvise_flags is union of dest_cid rather than other field?

Actually I am not sure whether union is appropriate in the Op struct. I avoid using it since it makes the code recondite.